qcodes/instrument_drivers/Spectrum/M4i.py

```diff
@@ -621,9 +621,13 @@ def initialize_channels(self, channels=None, mV_range=1000, input_path=0,
             mV_range, input_path, termination, coupling, compensation: passed
                 to the set_channel_settings function
             memsize (int): memory size to use for simple channel readout
+            pretrigger_memsize (None or int): pretrigger memory size to use. If None calculate a default value

```

Comment:
The ".. calculate a default value" part is misleading because nothing is being calculated, the default value is just hardcoded. I recommend to remove this notion. It is much more useful to add the fact that "16" is apparently the smallest possible value for pretrigger memory - this will be very helpful for users.